en/docs/learn/api-security/oauth2/grant-types/jwt-grant.md

```diff
@@ -1,69 +1,137 @@
+    ```
+
+    ``` java tab="Example"
+    curl -k -d "grant_type=password&username=admin&password=admin" -H "Authorization: Basic cEJ6dUlaaEdwaGZRbWRjVVgwbG5lRmlpdXh3YTo0U0pnV19qTU56aGpIU284OGJuZVhtTnFNMjRh" -H "Content-Type: application/x-www-form-urlencoded" https://localhost:9444/oauth2/token
+    ```
+
+    **Sample Response**
+    ```
+    {"access_token":"eyJ4NXQiOiJNell4TW1Ga09HWXdNV0kwWldObU5EY3hOR1l3WW1NNFpUQTNNV0kyTkRBelpHUXpOR00wWkdSbE5qSmtPREZrWkRSaU9URmtNV0ZoTXpVMlpHVmxOZyIsImtpZCI6Ik16WXhNbUZrT0dZd01XSTBaV05tTkRjeE5HWXdZbU00WlRBM01XSTJOREF6WkdRek5HTTBaR1JsTmpKa09ERmtaRFJpT1RGa01XRmhNelUyWkdWbE5nX1JTMjU2IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJhZG1pbiIsImF1dCI6IkFQUExJQ0FUSU9OX1VTRVIiLCJhdWQiOiJxNVlVSzloY1lMcUZXenZaQVJHb21UNzh6NThhIiwibmJmIjoxNjEzMTMwNTAxLCJhenAiOiJxNVlVSzloY1lMcUZXenZaQVJHb21UNzh6NThhIiwiaXNzIjoiaHR0cHM6XC9cL2xvY2FsaG9zdDo5NDQ0XC9vYXV0aDJcL3Rva2VuIiwiZXhwIjoxNjEzMTM0MTAxLCJpYXQiOjE2MTMxMzA1MDEsImp0aSI6IjdmZDc0NGQ5LWJlYTMtNDJiNS1hY2RjLWVmMDU0NGRjMmZhZiJ9.mBh0FB8kPKwkqP8rV78YXaSqmv_j-7kSKlylOfH8i1M3g8JvEQRx5gsxwMNidOr28DIL9hNv7Ebz6j6tqu-HUXtB_QgQGbF1QRALtBcYaV1-O8V_TAs02P7KYx_jfZv2HCVEE-s-jW2vBAMa24ZwgI0uCk4QHsev1chLd85FZdwbzqe0E0lt5hfB0Y5kLnZmzQFdBxkrv9eTppO5Z2WLyKFt9j2oI2a2eqa_NOYr7pwLyuzg-QL4Xz3SdfC3SMHPjAmaksJfcOS-SuswQ_edPN02fXVR0SKGCVIS-86u-9D4xgFvf_jtIg2Xd2LNNHY6BPBj1l9Nk_rfesnllx04ZA","refresh_token":"b6549344-c5ba-3103-8f93-475aa5416759","token_type":"Bearer","expires_in":3600}
+    ```
+
+Now you have obtained a JWT from WSO2 IS. In the next step, let us register an identity provider and create a service provider in WSO2 API-M.
+
+### Step 2 - Configure an Identity Provider and a Service Provider in WSO2 API-M
+
+1. Make sure WSO2 API-M is [up and running]({{base_path}}/install-and-setup/installation-guide/running-the-product/#starting-the-server).
+
+2. Sign in to the WSO2 API-M Management Console (`https://<API-M_Server_Host>:9443/carbon`)   
+
+3. Navigate to the **Identity Providers** section in the **Main** -> **Identity** section and click **Add.**
+
+4. Provide the following values to configure the Identity Provider.
+    -   **Identity Provider Name :** Enter the issuer name as the identity provider name. This is used to generate the JWT assertion. The issuer of a JWT generated by WSO2 IS is by default `https://localhost:9444/oauth2/token`.
+    -   **Identity Provider Public Certificate :** Choose the radio button named **Upload IDP certificate** so that the field named **Identity Provider Public Certificate** will be enabled. This is the certificate which will be used to sign the JWT assertion.
+
+        !!! info
+            The **Identity Provider Public Certificate** is the public certificate belonging to the identity provider. Uploading this is necessary to authenticate the response from the identity provider.
+
+            This can be any certificate. If the identity provider is another API Manager or an Identity Server, this can be a `wso2.crt` file. 
```

Comment:
cert file name can be anything

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3170/commits/d99c31c64db63b3df9a1aad0bc9edde2c10379f0